build.js

```diff
@@ -44,12 +44,14 @@ const compileCustomTest = (code, format = true) => {
         return `throw 'Test is malformed: ${match} is an invalid reference';`;
       }
       let importcode = compileCustomTest(customTests.api[name].__base, false);
+      const callback = importcode.includes('callback');
 
       importcode = importcode
           .replace(/var (instance|promise)/g, `var ${instancevar}`)
+          .replace(/callback\(/g, `${instancevar}(`)
```

Comment:
This looks strange. If the test code has `callback(something)`, does it make sense to replace it with `instance(something)`? Maybe it does, but then the variable name `instancevar` is no longer accurate and rather misleading.